### DISCUSSION TITLE
A sneak peak at something I'm working on for Version 1.0 Beta 4

### DISCUSSION THREAD

**Discussion by TalonFloof** [Talon396/EnhancedWeather#11] [Announcements]
I'm working on the radar for Version 1.0 Beta 4.
Here is a sneak peak of what I've got so far:

[Radar (WIP).webm](https://user-images.githubusercontent.com/105325988/196057368-156bd259-97ee-48f7-aa37-161c9c2561a6.webm)

As you can see, it's very different from the weather2 radar.

**Comment by Steveplays28** [comment]
That's super cool, nice!

**Reply by TalonFloof** [reply]
Thanks! It took a while to make this, though the video that you're seeing is a bit outdated as the radar has been improved since I made that video 5 days ago.

**Comment by TalonFloof** [comment]
@Steveplays28
Here's a screenshot of the current radar stuff.
![2022-10-20_21 58 50](https://user-images.githubusercontent.com/105325988/197291865-b390fe26-ecd0-4ece-a283-560ec89f3a6c.png)

There's also three tiers of radars, here's a screenshot of that.
![2022-10-21_16 36 29](https://user-images.githubusercontent.com/105325988/197292550-7a205d64-9979-4031-8003-e4ac342d4a6c.png)

**Reply by Steveplays28** [reply]
Awesome work, looks great!

May I suggest integration with Xaero's Minimap/Journeymap for the radar?
Might be cool to have a Minimap overlay for the weather, like you have for topology/day/night/biomes in those minimap mods

**Reply by TalonFloof** [reply]
Hmm, I could do that, though I would have to look into the APIs for Xaero's Minimap/Journeymap, if they even have any.

**Reply by Steveplays28** [reply]
Yeah. Just found out that Xaero's Minimap does not seem to have an API, or it's not obvious, but [JourneyMap has an API](https://github.com/TeamJM/journeymap-api)

The only mod that is compatible with Xaero's Minimap is [Open Parties and Claims](https://github.com/thexaero/open-parties-and-claims), because Xaero's Minimap references *their* API, not the other way around.

**Reply by TalonFloof** [reply]
Journeymap is a Forge mod though...

Also yes, Xaero's Minimap doesn't have an API. The only way to add an overlay would be to use Mixins.

**Reply by Steveplays28** [reply]
It has a Fabric version, I'm running it on 1.19.2

Also [Advanced Compass](https://www.curseforge.com/minecraft/mc-mods/advanced-compass) references Xaero's Minimap API but it seems to be closed source.

**Reply by Steveplays28** [reply]
Yeah Xaero's Minimap not having an API and Advanced Compass being closed source is not nice.

**Reply by TalonFloof** [reply]
> It has a Fabric version, I'm running it on 1.19.2

Oops, must have not looked hard enough at the versions on CurseForge. >_<
I'll look into if I can add an overlay to JourneyMap.

**Reply by Steveplays28** [reply]
CurseForge kekw, protip: you can just go to [Modrinth ](https://modrinth.com)and filter by modloader/version. Most Fabric/Quilt mods are on Modrinth instead of CurseForge.

Thank you! I appreciate that  
I'll open a proper issue for this
